### DISCUSSION TITLE
Welcome to Network Discussions!

### DISCUSSION THREAD

**Discussion by tomekmarchi** [universalweb/Network#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shahahmad** [comment]
Hey folks, name is Ahmad, I am a Software/Data Engineer specializing in Machine Learning. Have been following Sentivate for just over 2 years now, using the UW as my gateway into learning more about web development!

**Reply by tomekmarchi** [reply]
Welcome, you have a long road ahead but it'll teach you everything about how the Web works and how to fix it, code included.

**Comment by shahahmad** [comment]
@tomekmarchi there are a couple forks of the Sentivate Alpha Network on public GitHub (haven't been updated in 1-2 years), but I noticed the main repo isn't public yet. Not sure if these need a clean up. I.e. https://github.com/jmrandrsn/SentivateAlphaNetwork

**Reply by tomekmarchi** [reply]
Folks are free to fork whichever code they like so it's up to them if they want to keep a fork in their github etc. The code is public and the main repo is public being https://github.com/universalweb/Network/. That forked code is from a much older public repo of ours so no issue there. The older repo has been archived and was on the sentivate profile rather than the organization which is the current public main repo.

**Reply by tomekmarchi** [reply]
The fork on that user's account is also just a direct clone so no other code was added it looks like. Meaning they forked a repo of ours and it hasn't changed since on their account. Our old repo was at https://github.com/sentivate which is what that was forked from instead of the organization/community repo that is https://github.com/universalweb/

**Reply by tomekmarchi** [reply]
If you want to intern with us email me.
